tests/benchmark/README.md

```diff
@@ -0,0 +1,38 @@
+# BenchMark tests
+
+Following commands help run benchmark tests against 100 apis with 3
+versions/api. Every version has one api spec.
+
+```
+    export ITERATION_SIZE=10
+    export REGISTRY_PROJECT_IDENTIFIER=bench
+    export API_VERSIONS_COUNT=3
+    apg admin create-project --project_id=$REGISTRY_PROJECT_IDENTIFIER
+
+    go test --bench=Create ./tests/benchmark \
+        --registry_project=$REGISTRY_PROJECT_IDENTIFIER \
+        --version_count=$API_VERSIONS_COUNT \
+        --benchtime=${ITERATION_SIZE}x --timeout=0
```

Comment:
I was initially puzzled by this and subsequent invocations, wondering what was actually getting called for each value of the `--bench` argument. I found that it's treated as a regex that is used to match benchmark test function names, and since none of the test function names include "Create", "Update", etc., only the `--bench=List` invocation below calls an actual test function. That's probably because this was "design" for an implementation that is coming, but at least for clarity in review, I think it would be good to have the README instructions match the implementation.